edxval/migrations/0003_delete_transcriptcredentials.py

```diff
@@ -0,0 +1,16 @@
+# Generated by Django 2.2.12 on 2020-05-18 11:32
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('edxval', '0002_add_error_description_field'),
+    ]
+
+    operations = [
+        migrations.DeleteModel(
+            name='TranscriptCredentials',
+        ),
```

Comment:
Do you need to do this in two PRs, one for the code change and one for the migration?

Also, could we add a reverse migration to put this back just in case / to follow best practices?

Reply:
I will update the PR to separate code changes and migration changes